### ISSUE TITLE
References: ChatGPT Conversations

### ISSUE THREAD

**Issue by DiogenesAnalytics** [DiogenesAnalytics/autoencoder#22]
# About
What follows is simply a list of `ChatGPT` conversations on various topics related to autoencoders and general ML/DL topics (e.g. *model training*, *memory issues*, etc ...)

# Conversations
+ [Loss and val_loss explanation](https://chat.openai.com/share/16dbaae7-18c9-4b55-aeb8-955d554bf0d6)
+ [GPU Memory Error Fix](https://chat.openai.com/share/ac46ea8f-27dc-4696-8559-14979d72ff4a)
+ [Image Resize: 16:9 Standard](https://chat.openai.com/share/4aadea96-0bdf-4064-9f5b-f62aa8928b31)
+ [Shapes Mismatch: Resolve Error](https://chat.openai.com/share/9baef4e9-c47c-4a92-9b7a-1ab9631db1af)
+ [Autoencoder ABC in Python](https://chat.openai.com/share/89ec14f1-c047-4f85-bc91-2710d75f1a50)
+ [Optimal Batch Size Calculation](https://chat.openai.com/share/3c8594bc-c724-4283-bf2e-ce469456eeb5)
+ [Autoencoder Architecture Overview](https://chat.openai.com/share/ebc396fc-ad7b-4473-b9b7-6b38f2194130)
+ [Normalize Pixels for Neural Networks](https://chat.openai.com/share/e346d72e-af25-4f54-8a4a-8bf3b8121c8a)
+ [Optimal Epoch Search](https://chat.openai.com/share/4a7dbc00-bdc0-4fbe-a538-d05a44d8f0e7)
+ [Keras Learning Rate Scheduler](https://chat.openai.com/share/29ef4a37-410d-4a38-b851-3793e41b6d6e)
+ [Memory Profiling for Keras](https://chat.openai.com/share/15762821-29f3-4d73-855b-267044925082)
+ [Evaluate Keras Model](https://chat.openai.com/share/44eb6257-46e5-42d2-85a9-785b3fb1a42c)
+ [Iterative Training for Models](https://chat.openai.com/share/3fb4ad66-5f5f-4baa-8f94-176074dd5919)
+ [Extract Paths from TensorFlow Dataset](https://chat.openai.com/share/9c7e72ab-8612-44a7-873c-b72fca25f85a)
+ [Individual Image Evaluation Setup](https://chat.openai.com/share/f8464de2-8e42-4f40-8cf5-bfc7cff83518)
+ [Training Loss > Validation Loss](https://chat.openai.com/share/fba22203-a168-46a1-90eb-4559bbe29ede)
+ [Best Autoencoder for Images](https://chat.openai.com/share/7e73bfc0-ba04-4493-94bc-3271a45e62a6)
+ [Scaling Convolutional Layer Parameters](https://chat.openai.com/share/a7016125-f87a-45a7-9066-0fdeb64e73ec)
+ [Improve Human Body AE](https://chat.openai.com/share/2562a317-31fd-4cb2-b778-e9bfea5d4a0c)
+ [Autoencoder Threshold Calculation](https://chat.openai.com/share/4d773bdb-ee66-481b-8fd6-73faabc99ace)
+ [VGG16 Autoencoder with Keras](https://chat.openai.com/share/6fe4c689-742b-49cb-a6ef-c99ce205ccaa)
+ [CNN Layers: MaxPool & UpSample](https://chat.openai.com/share/04099568-6e62-43e1-9833-34b9864e64c2)
+ [Transfer Learning Overview](https://chat.openai.com/share/ae5ddf9d-8c3c-4678-b59a-c849ada2aff5)
+ [Autoencoder Performance Evaluation](https://chat.openai.com/share/a820a256-03b8-45b5-9583-720a6c58f568)
+ [Autoencoder to Classifier Tutorial](https://chat.openai.com/share/ece0087e-95cd-4f39-98b5-aa4cad15397f)
+ [Keras Dummy Model](https://chat.openai.com/share/a965e1d0-324e-42f3-8068-0b6e758e18ed)
+ [Visualizing Neural Network Features](https://chat.openai.com/share/0e52468c-a2d7-400b-8fb3-50962f65fa11)
+ [Visualizing Image Features: Python](https://chat.openai.com/share/d25723fb-5e3a-4a37-996d-d00cc4924d51)
+ [Image Clustering with Features](https://chat.openai.com/share/e53ac475-e790-4337-b083-6040bf0a304f)
+ [Evolutionary Algorithms for NN.](https://chat.openai.com/share/28dae1b2-2631-4651-a629-1b0b585755d4)
